main.go

```diff
@@ -169,12 +169,12 @@ func main() {
 		dir, _ := path.Split(cfg.VppAPISocket)
 		statsOpts = append(statsOpts, stats.WithSocket(path.Join(dir, "stats.sock")))
 		cleanupOpts = append(cleanupOpts, cleanup.WithDoneChan(cleanupDoneCh))
-
 		log.FromContext(ctx).Info("external vpp is being used")
-	} else { // If we don't have a VPPAPISocket, start VPP and use that
-		if err = cfg.VppInit.Decode("AF_PACKET"); err != nil {
+
+		if err = cfg.VppInit.Decode("NONE"); err != nil {
```

Comment:
I see that this `if branch` comes from the main. But the decode looks unnecessary here. I think we should do decode once on env config decoding.

Reply:
This `Decode("NONE")` was added for Calico-VPP.
Since forwarder uses an external VppAPISocket, Calico already created a management interface and we don't need to do it again.